roles/delete_nodepools/tasks/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+###################### Destroy Nodepools ##########################################################
+#Using the Ansible module ionoscloudsdk.ionoscloud.k8s_nodepool to destroy a node pool, it's important
+#to note that the module will perform the deletion whether you run it with or without the -C flag.
```

Comment:
i think with the introduction of the condition with the check mode, this comment needs to get updated as well :)